graylog2-web-interface/src/components/field-analyzers/FieldStatistics.jsx

```diff
@@ -159,28 +173,34 @@ const FieldStatistics = createReactClass({
   },
 
   _getHeaderCaret(column) {
-    if (this.state.sortBy !== column) {
+    const { sortBy, sortDescending } = this.state;
+    if (sortBy !== column) {
       return null;
     }
-    return this.state.sortDescending ? <i className="fa fa-caret-down" /> : <i className="fa fa-caret-up" />;
+    return sortDescending ? <i className="fa fa-caret-down" /> : <i className="fa fa-caret-up" />;
```

Comment:
Could potentially do this a little shorter.

```suggestion
    return <i className=`fa fa-caret-${sortDescending ? 'down' : 'up'}` />;
```